collector/collector.cpp

```diff
@@ -96,6 +96,25 @@ static void AbortHandler(int signum) {
   raise(signum);
 }
 
+static int __read(int fd, void* buf, int buflen) {
```

Comment:
Use of `_` as the first character in a global function is reserved by the compiler and could potentially lead to a conflict somewhere down the line. Could we rename it to something like `read_module` instead?

@stackrox/collector-team, we might want to consider moving module loading and unloading to a separate file in a follow up, it's starting to take up quite a bit of room here.